odl/operator/operator.py

```diff
@@ -206,29 +395,64 @@ class Operator(with_metaclass(_OperatorMeta, object)):
     implementation of the respective other is provided.
     """
 
+    def __new__(cls, *args, **kwargs):
+        """Create a new instance."""
+        instance = super().__new__(cls)
+
+        call_has_out, call_out_optional, _ = _dispatch_call_args(cls)
+        instance._call_has_out = call_has_out
+        instance._call_out_optional = call_out_optional
+        if not call_has_out:
+            # Out-of-place _call
+            instance._call_in_place = partial(_default_call_in_place,
```

Comment:
If I remember right, the main reason for this was the functions. If you initialize a `FunctionSetVector` with some externally given function, you cannot assign that directly to `_call` since it may not be inspectable (e.g. a NumPy `ufunc`). But you're right, one could wrap it into a new function with an optional `out` parameter in any case, would be simpler.